client/src/components/HomePage.jsx

```diff
@@ -0,0 +1,23 @@
+import ListLatestEntries from './ListLatestEntries'
+
+export default function HomePage() {
+
+    //fetch async function to db where entries are
+
+  return (
+    <div>
+        <br />
+
+        <img 
+        src="https://i.etsystatic.com/9472596/r/il/749ea1/3821016144/il_1588xN.3821016144_csuf.jpg" 
+        alt='bouquet of flowers' 
+        style={{ width: '200px', height: 'auto' }}
+        />
```

Comment:
```suggestion
        <img 
            src="https://i.etsystatic.com/9472596/r/il/749ea1/3821016144/il_1588xN.3821016144_csuf.jpg" 
            alt='bouquet of flowers' 
            style={{ width: '200px', height: 'auto' }}
        />
```

Nit comment, but it will help your overall experience with your code editor too. 
For this project, I'd install `prettier` code formatter. You probably don't need to change the settings but turn on the 'format on save' functionality so that it automatically adjusts your code/indentation etc. automatically.

Reply:
Thank you!  I installed prettier!